### DISCUSSION TITLE
[资讯]VP被元气桌面闪击

### DISCUSSION THREAD

**Discussion by Dave-12138** [share121/inter-knot#2549] [灌水]
![截图](https://s2.loli.net/2024/08/17/m7e9zXUEV18urhp.webp)
近日，@VacuolePao 在安装搜狗输入法时不慎被元气桌面偷袭，目前状况仍在僵持中。
据悉，VP对元气桌面使用了冻结，使其陷入挂起状态，暂时遏制了元气桌面的进攻。
<img src="https://upload-bbs.mihoyo.com/upload/2022/09/30/84a2430f8b496e493a5f131cc108ce8b_4422585394873060336.png" alt="米仔_瞅瞅" width="75" >

**Comment by HKSKurumi** [comment]
已阅，卸载处理（bushi）
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
